tests/core/propertyTypes.test.js

```diff
@@ -62,4 +62,19 @@ suite('propertyTypes', function () {
       assert.equal(stringify(this.el), '#hello');
     });
   });
+
+  suite('array', function() {
+    var parse = propertyTypes.array.parse;
+    var stringify = propertyTypes.array.stringify;
+
+    test('parses array', function () {
+      assert.deepEqual(parse(''), []);
+      assert.deepEqual(parse('5 test'), ['5', 'test']);
+    });
+
+    test('stringifies array', function () {
+      assert.equal(stringify([]), '');
+      assert.equal(stringify([5, 10]), '5 10');
```

Comment:
We already have a vector type: for instance ```lookAt: 0 0 0``` I would use coma to differentiate between vector and arrays. Also we might want to have arrays of vectors ```0 1 0, 0 3 0, -1 0 0, ...```